tools/shared/src/main/scala/org/scalajs/core/tools/sem/Semantics.scala

```diff
@@ -44,6 +49,7 @@ final class Semantics private (
     import scala.util.hashing.MurmurHash3._
     var acc = HashSeed
     acc = mix(acc, asInstanceOfs.hashCode)
+    acc = mix(acc, moduleInit.hashCode)
     acc = mixLast(acc, strictFloats.##)
     finalizeHash(acc, 1)
```

Comment:
1 should 3 (apparently we already failed to turn it to 2 when we added `strictFloats`).